src/renderer/actions/install.js

```diff
@@ -21,20 +22,16 @@ const DOWNLOAD_PACKAGE_PROGRESS = 60;
 const INJECT_SHIM_PROGRESS = 90;
 const RESTART_DISCORD_PROGRESS = 100;
 
-const oldBDFolder = path.join(remote.app.getPath("home"), "Library", "Preferences", "betterdiscord"); // Old MacOS
-const bdFolder = path.join(remote.app.getPath("appData"), "BetterDiscord");
-const bdDataFolder = path.join(bdFolder, "data");
-const bdPluginsFolder = path.join(bdFolder, "plugins");
-const bdThemesFolder = path.join(bdFolder, "themes");
 
-async function checkOldMacOS(folder) {
+async function checkOldBDFolder(folder) {
     if (await exists(folder)) {
-        log(`⚠️ Found old BD installation: ${folder}`);
+        log(`⚠️ Found old BD installation: ${folder}.`);
```

Comment:
Please don't add `.` to the end of these, they are intentionally omitted.

Reply:
This was done because it makes the GUI's text output nicer to read. The GUI's log now looks better. Like actual sentences, rather than weirdly cut off lines where the word-wrapping sometimes made it hard to see where one line ended and another began.